var/spack/repos/builtin/packages/dune-common/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class DuneCommon(PythonPackage):
+    """Dune, the distributed and unified numerics environment."""
+
+    homepage = "https://www.dune-project.org/doc/gettingstarted/"
+    git = "https://gitlab.dune-project.org/core/dune-common"
+    pypi = "dune-common/dune-common-2.9.0.tar.gz"
+    url = "https://files.pythonhosted.org/packages/c4/3e/48a506e9f42e1c3f384f55f578b887bba77916cb51f7ceff9d0bd9f315a8/dune-common-2.9.0.tar.gz"
```

Comment:
This property isn't needed since you're setting `pypi`.

(See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#id2.)